### ISSUE TITLE
add content for here's what we know page

### ISSUE THREAD

**Issue by ashao56** [code4policy/2024-a2-gun-violence#47]

**Comment by louis-costa** [comment]
Initial proposed page: 

January 5th Update from the Iowa Department of Public Safety
Location of Shooting:
Perry, IA Middle School
https://www.perry.k12.ia.us/

Perry is a town of 8,000 about 40 miles northwest of Des Moines

Summary of Events:
The events took place in the morning on Thursday, January 4th
[…]
Principal Dan Marburger placed himself in harm's way in an apparent effort to protect his students.

Outcome - As of Now
One victim: 11 year-old Ahmir Jolliff
The shooter, 17 year-old Dylan Butler, died from a self-inflicted gunshot wound
Seven other individuals received wounds or injuries of varying degree during the shooting event:
The school's principal and two other students who remain hospitalized
Four students who were released from the hospital.
Investigations:
Background investigations, as well as eyewitness accounts and victim interviews, are continuing.
Once the investigation is complete, the DCI’s investigative report will be turned over to the Dallas County Attorney’s Office to determine what additional course of action, if any, should be undertaken.
No further information will be released at this time.
Source: [Iowa Department of Public Safety (DPS)](https://dps.iowa.gov/update-perry-high-school-shooting-investigation)

[Home Page](http://localhost:8000/index.html)

**Comment by louis-costa** [comment]
Comment by Gal: would be nice to have pictures as well
